tests/pytests/test_multibyte_char_terms.py

```diff
@@ -1434,3 +1441,92 @@ def testTagSearch(env):
 
     res = conn.execute_command('FT.SEARCH', 'idx', '@t:{fussball}', 'NOCONTENT')
     env.assertEqual(res, [1, 'doc:2s'])
+
+def testLongTags(env):
+    env.cmd('FT.CREATE', 'idx', 'SCHEMA', 't', 'TAG')
+    conn = getConnectionByEnv(env)
+
+    t = unescaped_long_term
+    t_lower = t.lower()
+    t_escaped = escaped_long_term
+    t_escaped_lower = t_escaped.lower()
+
+    env.expect('HSET', '{doc}:1', 't', t).equal(1)
+    if not env.isCluster():
+        res = env.cmd(debug_cmd(), 'DUMP_TAGIDX', 'idx', 't')
+        # The conversion to lowercase occupies more space than the original
+        # term, so the term is stored in its original form
+        env.assertEqual(res, [[t, [1]]])
+
+    env.expect('HSET', '{doc}:2', 't', t_lower).equal(1)
+    if not env.isCluster():
+        res = env.cmd(debug_cmd(), 'DUMP_TAGIDX', 'idx', 't')
```

Comment:
Once reallocation is enabled, `DUMP_TAGIDX` will return only one term and the assert is expected to fail, right?

Reply:
cool
maybe add a small comment as a reminder

Reply:
OK. Done.